src/System.Net.HttpListener/tests/HttpListenerContextTests.cs

```diff
@@ -54,12 +62,64 @@ public static IEnumerable<object[]> SubProtocol_TestData()
         }
 
         [ConditionalFact(nameof(IsNotWindows7OrUapCore))]
-        public async Task AcceptWebSocketAsync_SocketSpoofingAsWebSocket_ThrowsWebSocketException()
+        public async Task AcceptWebSocketAsync_ValidWebSocket_SetsUpHeadersInResponse()
```

Comment:
Are you not seeing random failures in your new tests when running the managed implementation on Windows?